From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Filip Schauer <f.schauer@proxmox.com>
Date: Mon, 15 Dec 2025 15:59:31 +0100
Subject: [PATCH] start: Remove outdated comment about group dropping

Commit b58214ac30bd (tree-wide: improve setgroups() dropping) moved the
group dropping code to occur before lxc_switch_uid_gid. Therefore this
comment is no longer correct.

Signed-off-by: Filip Schauer <f.schauer@proxmox.com>
(cherry picked from commit e6ad17a796ae056337744980fb0719607a6be985)
Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 src/lxc/start.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/lxc/start.c b/src/lxc/start.c
index 3160459b4..e10fbfaed 100644
--- a/src/lxc/start.c
+++ b/src/lxc/start.c
@@ -1115,9 +1115,6 @@ static int do_start(void *data)
 		if (!handler->conf->root_nsgid_map)
 			nsgid = handler->conf->init_gid;
 
-		/* Drop groups only after we switched to a valid gid in the new
-		 * user namespace.
-		 */
 		if (!lxc_drop_groups() &&
 		    (handler->am_root || errno != EPERM))
 			goto out_warn_father;
